README.md

```diff
@@ -75,16 +103,14 @@ Admin endpoints are:
 
 `/__build-info`
 
-_These standard endpoints do not need to be specifically documented._
-
-_This section *should* however explain what checks are done to determine health and gtg status._
-
-There are several checks performed:
+There are several checks performed:  
+ 
+* Checks that DynamoDB table is accessible, using parameters supplied on service startup.  
 
-_e.g._
-* Checks that a connection can be made to Neo4j, using the neo4j url supplied as a parameter in service startup.
+See the api/api.yml for the swagger definitions of these endpoints  
 
 ## Other information
+TODO  
```

Comment:
Can I leave it for now. I may need to use this section when I actually start implementing the service.